Atomic/util.py

```diff
@@ -794,6 +798,67 @@ def load_scan_result_file(file_name):
     """
     return json.loads(open(os.path.join(file_name), "r").read())
 
+
+class InstallData(object):
+
+    @staticmethod
+    def read_install_data():
+        if os.path.exists(ATOMIC_INSTALL_JSON):
+            return json.loads(open(os.path.join(ATOMIC_INSTALL_JSON), "r").read())
+        return {}
+
+    @staticmethod
+    def write_install_data(install_data):
+        with open(ATOMIC_INSTALL_JSON, 'w') as write_data:
+            time_out = 0
+            f_lock = False
```

Comment:
The problem with separate files is it makes it significantly more complicated to look up images by hash, etc.  And eventually we come down to the same problem of reading or writing the same file.  I'll extend the locking.